scripts/userstyles.yml

```diff
@@ -695,3 +698,9 @@ userstyles:
       app-link: "https://nextjs.org"
       current-maintainers: [*Dandraghas]
 # yaml-language-server: $schema=https://raw.githubusercontent.com/catppuccin/userstyles/main/scripts/userstyles.schema.json
+  inoreader:
+    name: inoreader
+    category: productivity
+    readme:
+      app-link: "https://inoreader.com"
+      current-maintainers: [ *thomas-philippot ]
```

Comment:
can you place this section of code in alphabetical order like the rest of the themes.